docs/source/release/v6.8.0/Framework/Data_Handling/New_features/35935.rst

```diff
@@ -0,0 +1 @@
+* :ref:`LoadDiffCal <algm-LoadDiffCal>` now accepts an alternate grouping definition in the HDF format.
```

Comment:
Can you use a `-` instead of `*` for the bullet point? Using a `*` can cause issues further down the line for the Release Editors
```suggestion
- :ref:`LoadDiffCal <algm-LoadDiffCal>` now accepts an alternate grouping definition in the HDF format.
```